packages/ui/src/molecules/PaymentMethods/PaymentMethods.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render } from '@testing-library/react'
+import { axe } from 'jest-axe'
+import React from 'react'
+
+import PaymentMethods from './PaymentMethods'
+
+describe('Payment methods', () => {
+  it('should have `data-store-payment-methods` attribute', () => {
```

Comment:
What do you think about creating tests for all other data attributes?

Reply:
Sure! I gonna do that.